Formula/v/vci.rb

```diff
@@ -0,0 +1,35 @@
+class Vci < Formula
+  desc "VulnCheck command-line tool"
+  homepage "https://github.com/vulncheck-oss/cli"
+  url "https://github.com/vulncheck-oss/cli/archive/refs/tags/v0.4.2.tar.gz"
+  sha256 "d06798ef8321c08835c9370fbe4545ecf0eb5c9b21cbc7db3f6e090ce34fafd7"
+  license "Apache-2.0"
+
+  head "https://github.com/vulncheck-oss/cli.git", branch: "main"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "go" => :build
+
+  deny_network_access! [:postinstall, :test]
+
+  def install
+    with_env(
+      "VC_VERSION" => version.to_s,
+      "GO_LDFLAGS" => "-s -w -X main.updaterEnabled=vulncheck-oss/cli",
```

Comment:
what is the ref for `updaterEnabled`?

Reply:
This is a reference/placeholder for a future feature where the cli will notify the user if it's out of date in some certain situations

Reply:
we dont allow self-update though